src/froll.c

```diff
@@ -669,3 +683,390 @@ void frollmaxExact(double *x, uint64_t nx, ans_t *ans, int k, double fill, bool
     }
   }
 }
+
+static inline void wmin(const double * restrict x, uint64_t o, int k, double * restrict w, uint64_t *iw, bool narm) {
```

Comment:
not a big fan of macros but couldn't we do something like and deduplicate `wmin` and `wmax`?
```c
#define DEFINE_WINDOWFUN(fname, init_val, op)
DEFINE_WINDOWFUN(wmax, R_NegInf, >=)
DEFINE_WINDOWFUN(wmin, R_PosInf, <=)
```

Reply:
Same holds for `frollmaxFast` and `frollmaxExact`